tests/superphot_plus/plotting/test_plot_sampling_results.py

```diff
@@ -0,0 +1,167 @@
+import os, glob
+
+import numpy as np
+import pytest
+
+from superphot_plus.plotting.sampling_results import (
+    plot_posterior_hist_numpyro_dict,
+    plot_sampling_trace_numpyro,
+    plot_corner_plot_all,
+    compare_oversampling,
+    plot_combined_posterior_space,
+    plot_param_distributions,
+    plot_oversampling_1d,
+)
+
+
+def generate_dummy_posterior_sample_dict(batch=False):
```

Comment:
If you want to be fancy, you could turn this into a pytest fixture by adding it in the conftest file and putting a fixture annotation on top. Then, you can use it with lines like:

```
def test_plot_posterior_hist_numpyro_dict(tmp_path, dummy_posterior_sample_dict):
  ...
  plot_posterior_hist_numpyro_dict(dummy_posterior_sample_dict, parameter, tmp_path)
```

You'd likely want one version that's plain, and one that's batched.